src/components/App.vue

```diff
@@ -131,6 +135,8 @@
             :theme="theme"
             :systemTheme="systemTheme"
             :allowBetaVersions="settings.allowBetaVersions"
+            :createdTime="createdTime"
```

Comment:
This doesn't currently show the `createdTime`, so these props could be removed.

If you want to show the `createdTime` somewhere, then the prop could be kept